ui_panels.py

```diff
@@ -2617,38 +2673,155 @@ def draw_properties(self, layout, width=250):
+            is_free = self.asset_data.get("isFree")
+
+            # Get pricing info from extensions cache
+            is_for_sale, base_price = asset_bar_op.get_addon_pricing_data(
+                self.asset_data
             )
+
+            if self.asset_data["isPrivate"]:
+                t = "Private"
+                self.draw_property(box, "Access", t, icon="LOCKED")
+            elif is_for_sale and not can_download and base_price:
+                t = f"${base_price} (Not purchased)"
+                icon = pcoll["full"]
+                self.draw_property(
+                    box,
+                    "Price",
+                    t,
+                    icon_value=icon.icon_id,
+                    tooltip="This addon is for sale but you haven't purchased it yet",
+                )
+            elif is_for_sale and can_download and base_price:
+                t = f"${base_price} (Purchased)"
+                icon = pcoll["full"]
+                self.draw_property(
+                    box,
+                    "Price",
+                    t,
+                    icon_value=icon.icon_id,
+                    tooltip="You have purchased this addon",
+                )
+            elif not is_free and not is_for_sale:
+                t = "Full plan required"
+                icon = pcoll["full"]
+                self.draw_property(
+                    box,
+                    "Access",
+                    t,
+                    icon_value=icon.icon_id,
+                    tooltip=plans_tooltip,
+                    url=paths.BLENDERKIT_PLANS_URL,
+                )
+            else:
+                t = "Free"
+                icon = pcoll["free"]
+                self.draw_property(
+                    box,
+                    "Access",
+                    t,
+                    icon_value=icon.icon_id,
+                    tooltip="This addon is free to use",
+                )
+
+            # Display Blender version requirements for addons
+            dict_params = self.asset_data.get("dictParameters", {})
+            min_version = dict_params.get("blenderVersionMin")
+            max_version = dict_params.get("blenderVersionMax")
+
+            if min_version or max_version:
+                version_text = ""
+                if min_version and max_version:
+                    version_text = f"{min_version} - {max_version}"
+                elif min_version:
+                    version_text = f"{min_version}+"
+                elif max_version:
+                    version_text = f"≤ {max_version}"
+
+                # Check if current Blender version is compatible
+                current_version = (
+                    f"{bpy.app.version[0]}.{bpy.app.version[1]}.{bpy.app.version[2]}"
+                )
+                is_compatible = True
+
+                if min_version:
+                    try:
+                        current_tuple = tuple(map(int, current_version.split(".")))
```

Comment:
So the code first converts the version tupple into a string f"{bpy.app.version[0]}.{bpy.app.version[1]}.{bpy.app.version[2]}" and then parses it back into a tupple? Very elegant.